src/Illuminate/Foundation/Testing/Concerns/MocksApplicationServices.php

```diff
@@ -88,7 +89,14 @@ protected function withoutEvents()
             $this->firedEvents[] = $called;
         });
 
+        $mock->shouldReceive('until')->andReturnUsing(function ($called) {
+            $this->firedEvents[] = $called;
+
+            return true;
```

Comment:
That's something that should be tested on the event handlers themselves and these tests that are simply asserting that a given event should be fired shouldn't worry about the return of those possible events. That's something another test of the handler should cover.